### DISCUSSION TITLE
Daf-mobile setup difficulty

### DISCUSSION THREAD

**Discussion by pvanroos** [decentralized-identity/veramo#297] [Q&A] [answered]
Could not get Daf-mobile installed locally despite following instructions on both Mac and Windows (and trying for both iOS and Android).  I was able to get the Agent and Hub working with no problems.  Despite that the fact that I've built and deployed React Native apps to iPhone Android before, I still need some guidance here.

Is there an easier way to install on my iOS device (iPhone)?

**Comment by jasheal** [comment, marked as answer]
Hey Paul, We will be releasing some tutorials this week (along with the latest version of daf) to get started building in React Native from scratch. There’s a lot of outdated code in ‘daf-mobile’ right now and we most likely will create a new mobile reference implementation from scratch inline with the new docs minus all the experimental code. However, You should be able build this branch as I’ve been recently been working on it to ensure RN support for the latest version of daf. 

https://github.com/uport-project/daf-mobile/tree/feature/daf-7-upgrade

Let us know if you have any issues?

**Reply by trentlarson** [reply]
Thanks for this pointer!  FYI, I was able to run this app on master branch locally for iOS (even though I can't get react-native Veramo to work yet).

**Comment by pvanroos** [comment]
Great.  I gave up yesterday but I’ll check this out.

 

Actually, the swagger definitions in the Agent are extremely helpful in divining the potential of this system. 

 

-Paul
